search-sentiments-map/app.js

```diff
@@ -0,0 +1,60 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+const PORT = process.env.PORT || 3000;
+const http = require('http');
+const fs = require('fs');
+const path = require('path');
```

Comment:
Consider using CONSTANT_CASE for all of these.

https://google.github.io/styleguide/jsguide.html#naming-constant-names

Reply:
got it

Reply:
What you did is not constant case. Constant case is HTTP, not Http.

Reply:
omg I read the wrong line lol

Reply:
No wait, this is supposed to be a "not constant" case. I just read it again.

Reply:
This is supposed to mirror the imported name

Reply:
Okay, makes sense!